docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -0,0 +1,2966 @@
+    "            {\n",
+    "                \"id\": \"Patient.name\",\n",
+    "                \"path\": \"Patient.name\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.telecom\",\n",
+    "                \"path\": \"Patient.telecom\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.address\",\n",
+    "                \"path\": \"Patient.address\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.photo\",\n",
+    "                \"path\": \"Patient.photo\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.contact\",\n",
+    "                \"path\": \"Patient.contact\",\n",
+    "                \"max\": \"0\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.gender\",\n",
+    "                \"path\": \"Patient.gender\",\n",
+    "                \"min\": 1\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.extension\",\n",
+    "                \"path\": \"Patient.extension\",\n",
+    "                \"slicing\": {\n",
+    "                    \"discriminator\": [\n",
+    "                        {\n",
+    "                            \"type\": \"value\",\n",
+    "                            \"path\": \"url\"\n",
+    "                        }\n",
+    "                    ],\n",
+    "                    \"ordered\": False,\n",
+    "                    \"rules\": \"open\"\n",
+    "                }\n",
+    "            },\n",
+    "            {\n",
+    "                \"id\": \"Patient.extension:patient-race\",\n",
+    "                \"path\": \"Patient.extension\",\n",
+    "                \"sliceName\": \"patient-race\",\n",
+    "                \"short\": \"Race\",\n",
+    "                \"definition\": \"Race of patient\",\n",
+    "                \"min\": 0,\n",
+    "                \"max\": \"1\",\n",
+    "                \"type\": [\n",
+    "                    {\n",
+    "                        \"code\": \"Extension\",\n",
+    "                        \"profile\": [\n",
+    "                            \"http://fhir.kids-first.io/StructureDefinition/patient-race\"\n",
+    "                        ]\n",
+    "                    }\n",
+    "                ],\n",
+    "                \"isModifier\": False\n",
+    "            }\n",
+    "        ]\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "#### Explanation of Parts\n",
+    "Let's break down each piece. First we'll start with the piece that says we want to use extensions in the `Patient` `StructureDefinition`:\n",
```

Comment:
I don't think the reason for this is explained. Can you add something that talks about what stops you from just adding 
```
{
    "id": "Patient.race",
    "path": "Patient.race",
    "min": 1,
    ...
 },
```
to the element list directly instead of adding this extra intermediate layer?

Reply:
Good point. It's because of https://www.hl7.org/fhir/profiling.html#limitations. I'll add something about this